### DISCUSSION TITLE
Cant get SimClr to meet memory requirements

### DISCUSSION THREAD

**Discussion by GeorgePearse** [open-mmlab/mmclassification#1663] [Q&A] [answered]
I've reduced the batch size to 1 and the crop size to 2 and yet SimCLR won't start training. 

Is there anything else I can reduce to reduce memory consumption? Or another SSL algorithm I should try?  Just trying to get things running on 1  a10 GPU for the time-being

**Comment by Ezra-Yu** [comment, marked as answer]
Maybe you can MOCOv2. SimClr and MOCOv3 all require large batch-size and usually need serval GPUs with lager graphics memory.

**Comment by GeorgePearse** [comment]
How large is large / what machines, or sets of GPUs are normally used as a minimum?
